streamlabs4j/src/main/java/com/github/twitch4j/streamlabs4j/api/domain/StreamlabsUser.java

```diff
@@ -151,7 +156,7 @@
         /**
          * Id of the user
          */
-        private Long id;
+        private String id;
```

Comment:
This is a breaking change so I'd either revert or have a `getId()` method that returns a Long to avoid the breaking change... then you can add `getStringId()` (or whatever name is best) to get the id without the parsing.

Reply:
To be honest, the returned value is a long in a string format, so unless jackson has some problems with it I don't see many reasons for this workaround, and probably shouldn't have pushed this change.
It looks like Facebook user ids are numeric strings according to their [API documentation](https://developers.facebook.com/docs/graph-api/reference/user/), so I doubt conflicts could come up in the future.